apps/server/src/modules/h5p-editor/uc/h5p.uc.ts

```diff
@@ -59,6 +61,8 @@ export class H5PEditorUc {
 	}
 
 	private mapH5pError(error: unknown) {
+		console.error(error);
```

Comment:
why just putting it to the console instead of proper error handling?

Reply:
Was left over from testing, removed